graphql_compiler/compiler/ir_lowering_sql/metadata.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018-present Kensho Technologies, LLC.
+from ... import exceptions
+from .constants import SqlBackend
+
+
+class SqlMetadata(object):
+    """Metadata wrapper for use during compilation.
+
+    In order to transform GraphQL to SQL, there needs to be additional configuration specified
+    for mapping:
+        - GraphQL types -> SQL tables
+        - GraphQL fields -> SQL columns
+        - GraphQL Edges -> SQL JOINs
```

Comment:
I'd probably lowercase `edges` here, for consistency with the previous two lines.